src/base/bittorrent/session.cpp

```diff
@@ -4545,18 +4549,15 @@ void Session::startUpTorrents()
                     resumeData.useAutoTMM = false;
                     resumeData.savePath = storageLocation;
                     resumeData.downloadPath = {};
-                    LogMsg(tr("Torrent '%1' is assigned the recovered category '%2' whose paths don't match the torrent's path."
-                              " Torrent is switched to \"Manual\" mode.").arg(torrentID.toString(), category), Log::WARNING);
+                    LogMsg(tr("Detected mismatch between save paths of the recovered category and the current save paths. Torrent is now switched to Manual mode."
```

Comment:
```suggestion
                    LogMsg(tr("Detected mismatch between save paths of the recovered category and the current save path of torrent. Torrent is now switched to Manual mode."
```
The current save path of torrent is the only one.

Reply:
> Then "save paths of the recovered category" should be "save path of the recovered category" right?

Category has two options: "regular" save path and download path (i.e. save path for incomplete torrents).